qubes/ext/services.py

```diff
@@ -18,34 +18,62 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library; if not, see <https://www.gnu.org/licenses/>.
 
-'''Extension responsible for qvm-service framework'''
+"""Extension responsible for qvm-service framework"""
 
+import os
 import qubes.ext
+import qubes.config
+
 
 class ServicesExtension(qubes.ext.Extension):
-    '''This extension export features with 'service.' prefix to QubesDB in
+    """This extension export features with 'service.' prefix to QubesDB in
     /qubes-service/ tree.
-    '''
+    """
+
+    @staticmethod
+    def add_dom0_services(vm, service):
```

Comment:
Since it takes just one service, I'd use singular `service` in the function name.